bin/kafka-producer-perf-test.sh

```diff
@@ -17,4 +17,4 @@
 if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
-exec $(dirname $0)/kafka-run-class.sh kafka.tools.ProducerPerformance $@
+exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.clients.tools.ProducerPerformance $@
```

Comment:
@junrao @ijuma  After doing some implementation, I felt argparse4j is more flexible than jopt-simple. I have used argparse4j  for parsing command line options. In future, we can migrate existing jopt usages to argparse4j.